src/core/jsonl/iterator.cc

```diff
@@ -1,7 +1,10 @@
+#include <istream>
```

Comment:
Please put this with the other system includes and add a comment to why we are requiring it